front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -157,7 +159,10 @@ export default function AssistantBuilder({
           generationSettings: initialBuilderState.generationSettings ?? {
             ...getDefaultAssistantState().generationSettings,
           },
-          actions: [], // Actions will be populated later from the client
+          actions: initialBuilderState.actions.map((action) => ({
```

Comment:
This is correct since we will have actions in initialBuilderState if only we are duplicating agents (and we should leave a comment about it)